tests/core/daemon/test_daemon.py

```diff
@@ -1107,3 +1134,470 @@ async def test_keyring_file_deleted(
+            "plotter": "bladebit",
+            "plot_type": "ramplot",
+            "w": True,
+            "m": True,
+            "no_cpu_affinity": True,
+            "k": 25,
+            "r": 2,
+            "u": 128,
+            "e": False,
+            "parallel": False,
+            "n": 1,
+            "queue": "default",
+            "d": "root_path",
+            "t": "root_path",
+            "t2": "",
+            "f": "",
+            "plotNFTContractAddr": "",
+            "x": False,
+            "b": 512,
+            "overrideK": True,
+            "delay": 0,
+            "a": 3598820529,
+            "c": "hello",
+        },
+        response={
+            "success": True,
+        },
+    ),
+    RouteCase(
+        route="start_plotting",
+        description="bladebit - diskplot",
+        request={
+            "service": "chia_plotter",
+            "plotter": "bladebit",
+            "plot_type": "diskplot",
+            "w": True,
+            "m": True,
+            "no_cpu_affinity": True,
+            "k": 25,
+            "r": 2,
+            "u": 128,
+            "e": False,
+            "parallel": False,
+            "n": 1,
+            "queue": "default",
+            "d": "root_path",
+            "t": "root_path",
+            "t2": "testing",
+            "f": "",
+            "plotNFTContractAddr": "",
+            "x": False,
+            "b": 512,
+            "overrideK": True,
+            "delay": 0,
+            "a": 3598820529,
+            "c": "hello",
+            "cache": "cache",
+            "f1_threads": 5,
+            "fp_threads": 6,
+            "c_threads": 4,
+            "p2_threads": 4,
+            "p3_threads": 4,
+            "alternate": True,
+            "no_t1_direct": True,
+            "no_t2_direct": True,
+        },
+        response={
+            "success": True,
+        },
+    ),
+    RouteCase(
+        route="start_plotting",
+        description="madmax",
+        request={
+            "service": "chia_plotter",
+            "plotter": "madmax",
+            "plot_type": "diskplot",
+            "w": True,
+            "m": True,
+            "no_cpu_affinity": True,
+            "k": 25,
+            "r": 2,
+            "u": 128,
+            "e": False,
+            "parallel": False,
+            "n": 1,
+            "queue": "default",
+            "d": "root_path",
+            "t": "root_path",
+            "t2": "testing",
+            "f": "",
+            "plotNFTContractAddr": "",
+            "x": False,
+            "b": 512,
+            "overrideK": True,
+            "delay": 0,
+            "a": 3598820529,
+            "c": "hello",
+            "v": 128,
+        },
+        response={
+            "success": True,
+        },
+    ),
+)
+@pytest.mark.asyncio
+async def test_plotter_options(
+    daemon_connection_and_temp_keychain: Tuple[aiohttp.ClientWebSocketResponse, Keychain],
+    get_b_tools: BlockTools,
```

Comment:
Isn't this just `b_tools` or `bt`?  It isn't a thing you call to get the tools.

Reply:
`get_b_tools` is a fixture in conftest.py